src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,32 @@
+package seedu.addressbook.data.tag;
+
+public class Tagging {
+	private char sign;
+	private String personName;
+	private String tag;
+	
+	public Tagging() {
```

Comment:
What is the point of this default constructor? Java has it by default right? Remove it.